pandas/tools/hashing.py

```diff
@@ -0,0 +1,18 @@
+import warnings
+import sys
+
+m = sys.modules['pandas.tools.hashing']
+for t in ['hash_pandas_object', 'hash_array']:
+
+    def outer(t=t):
+
+        def wrapper(*args, **kwargs):
+            from pandas import util
+            warnings.warn("pandas.tools.hashing is deprecated and will be "
+                          "removed in a future version, import "
+                          "from pandas.util",
+                          FutureWarning, stacklevel=3)
```

Comment:
Thoughts on making this one a DeprecationWarning? The only reason I ask is that dask did a release yesterday. Could do a point release fixing this if needed though.